examples/data/shader.frag

```diff
@@ -0,0 +1,27 @@
+#version 330
```

Comment:
Can you move this file next to the python file that is using it?
Alternatively rename it so it will be more example specific, for example: `glfw-simple-window.frag`

Reply:
As far as organization - do you think it's better to have shaders in a different folder tree?  I was thinking that any file that needed to be read and processed should be found within the same folder tree.  So I am leaning towards a rename.